fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,292 @@
+/datum/action/receding_stance/Trigger(mob/living/M, mob/living/user)
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't do stances while incapacitated...</span>")
+		return
+	if(stancing)
+		to_chat(owner, "<span class='warning'>You are already perfoming a stance.</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if(owner.mind.martial_art.streak == "receding_stance")
+		owner.visible_message("<span class='danger'>[owner] stops moving back.</i></b>")
+		owner.mind.martial_art.streak = ""
+	else
+		owner.visible_message("<span class='danger'>[owner] moves back and begins to form a stance.</span>", "<b><i>You backpedal and begin to form your stance.</i></b>")
+		stancing = TRUE
+		addtimer(VARSET_CALLBACK(src, stancing, FALSE), 30, TIMER_UNIQUE)
+		if(do_after(owner, 3 SECONDS))
+			owner.visible_message("<span class='danger'>[owner] focuses on his stance.</span>", "<b><i>You focus on your stance. Stamina...</i></b>")
+			owner.mind.martial_art.streak = "receding_stance"
+			H.adjustStaminaLoss(-40)
+		else
+			owner.visible_message("<span class='danger'>[owner] stops moving back.</i></b>")
+			return
+
+/datum/martial_art/velvetfu/proc/receding_stance(mob/living/carbon/user)
+	return // Empty return because we only want the button trigger
+
+/// Twisted Stance
+/datum/action/twisted_stance
+	name = "Twisted Stance - Regenerates a lot of stamina, deals brute damage."
+	icon_icon = 'fulp_modules/lisa/sprites/stances.dmi'
+	button_icon_state = "twisted_stance"
+
+/datum/action/twisted_stance/Trigger(mob/living/M, mob/living/user)
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't do stances while incapacitated...</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if(owner.mind.martial_art.streak == "twisted_stance")
+		owner.visible_message("<span class='danger'>[owner] untwists [owner.p_them()]self.</i></b>")
+		owner.mind.martial_art.streak = ""
+	else
+		owner.visible_message("<span class='danger'>[owner] suddenly twists and turns, what a strange stance!</span>", "<b>You twist and turn, your twisted stance is done!</b>")
+		owner.mind.martial_art.streak = "twisted_stance"
+		H.adjustStaminaLoss(-40)
+		H.apply_damage(18, BRUTE, BODY_ZONE_CHEST, wound_bonus = CANT_WOUND)
+
+/datum/martial_art/velvetfu/proc/twisted_stance(mob/living/carbon/user)
+	return // Empty return because we only want the button trigger
+
+
+/// Flying Axe Kick - Deals Brute and causes bleeding. Costs 50 Stamina.
+/datum/martial_art/velvetfu/proc/flyingAxekick(mob/living/A, mob/living/D)
+	log_combat(A, D, "flying kick (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] flying kicked [D], such skill!</span>", \
+					"<span class='userdanger'>Your neck is flying kicked by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You flying kick [D]!</span>")
+	A.adjustStaminaLoss(50)
+	if(prob(60))
```

Comment:
MM DON'T YOU LOVE IT WHEN YOU DEAL 50 POINTS OF STAMINA DAMAGE TO YOURSELF TO DO JACK SHIT TO YOUR TARGET BECAUSE YOU FAILED A 60% DIE ROLL, I KNOW I DO

Reply:
and yes, the 10 brute damage is still there, but isn't that the same damage the disarm does anyway?